examples/example1.html

```diff
@@ -51,15 +50,15 @@
 
         <script src="http://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
         <script type="text/javascript">
-            $(document).ready(function() {
-                $.ajax('js/example1.js', {dataType: 'script'})
-                    .always(function(response) {                        
-                        $("#code").text(response.responseText);
-                        $('pre code').each(function(i, block) {
-                            hljs.highlightBlock(block);
-                        });
-                    });
-            });
+           var oReq = new XMLHttpRequest();
+           oReq.addEventListener("load", (function(xhr) {
+             var code = xhr.currentTarget.response;
+             var codeBlock = document.getElementById("code");
+             codeBlock.innerHTML = code;
```

Comment:
I think we should use `innerText`. There are some HTML within the javascript code, and innerText will prevent to render it as HTML.

Reply:
changed it to `textContent` instead: `innerText` strips whitespace, which is no problem until the first comment :)